README.md

```diff
@@ -60,7 +60,7 @@ $ npm install
 ##### Running on localhost:
 * **Step 1:** Fork susper repository and clone it to your desktop
 * **Step 2:** Then cd into that cloned folder
-* **Step 3:** Then checkout to your master branch `$ git checkout angular`
+* **Step 3:** Then checkout to your master branch `$ git checkout master`
```

Comment:
Do you think we need this line? By default the branch will be master. No need to explicitly checkout to master.